File: 1051. Height Checker/solution.java
class Solution {
  public int heightChecker(int[] heights) {
    int[] sortedHeights = heights.clone();
    Arrays.sort(sortedHeights);
    int result = 0;
    for (int index = 0; index < heights.length; index++){
      if (sortedHeights[index] != heights[index]){
        result += 1;
      }
    }

    return result;
  }
}

File: 1051. Height Checker/solution.js
/**
 * @param {number[]} heights
 * @return {number}
 */
 var heightChecker = function(heights) {
  let sortedHeights = [...heights]
  sortedHeights.sort(function(a, b){return a-b});
  let result = 0;
  for (let index = 0; index < heights.length; index++){
    if (sortedHeights[index] !== heights[index]){
      result += 1;
    }
  }

  return result;
};